profile_library/signify/929002471901/model.json

```diff
@@ -0,0 +1,17 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2025-01-22T15:53:12.576838",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly PM mini Gen3",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.17.2:docker"
+  },
+  "aliases": [
+    "929002471901"
+  ],
```

Comment:
```suggestion
```

This can be removed, only have to be added when different than the directory name.